source/_integrations/zwave_me.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: Z-Wave.Me Z-Way
+description: Instructions on how to integrate Z-Wave with Home Assistant via Z-Wave.Me Z-Way.
+featured: true
+ha_category:
+  - Binary Sensor
+  - Climate
+  - Cover
+  - Fan
+  - Hub
+  - Light
+  - Lock
+  - Number
+  - Select
+  - Sensor
+  - Siren
+  - Switch
+ha_release: '2021.12'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@lawfulchaos, @Z-Wave-Me'
+ha_domain: zwave_me
+ha_platforms:
+  - binary_sensor
+  - climate
+  - cover
+  - fan
+  - light
+  - lock
+  - number
+  - sensor
+  - select
+  - siren
+  - switch
+---
+
+This integration allows you to control a Z-Wave network via the [Z-Wave.Me Z-Way](https://z-wave.me/z-way/). This is the recommended Z-Wave integration for Home Assistant for owners of Z-Wave.Me controllers.
```

Comment:
I don't think we should write that this is the recommended Z-Wave integration for Z-Wave.Me controllers. Home Assistant doesn't recommend (nor discommend) this integration.

You're welcome to write the benefits of this integration that other Z-Wave integrations may not provide.

Reply:
Well, for owners of ZME controllers (Z-Way software) this is the only one. But okay, we can rephrase it